src/products/dtos/product.dto.ts

```diff
@@ -0,0 +1,21 @@
+export class ProductDTO {
+  readonly name: string;
```

Comment:
Any of these properties have `class-validator` validations, hence, we can create products without any restriction. Remember to add those in each DTO that you are creating :)

Reply:
I remove the DTOs on this PR because I'm not longer using it. In later PRs, I will use `class-validator` for sure

Reply:
Nice, thanks